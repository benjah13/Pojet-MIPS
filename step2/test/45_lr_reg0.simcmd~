#Commande a lancer dans le script 45_lr_reg0

lr bash 0xff
