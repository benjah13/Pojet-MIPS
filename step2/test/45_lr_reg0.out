The registre $0 cannot be modified!
